circuit-codegen/src/main/kotlin/com/slack/circuit/codegen/CircuitSymbolProcessorProvider.kt

```diff
@@ -103,21 +128,27 @@ private class CircuitSymbolProcessor(
     return emptyList()
   }
 
+  @OptIn(KspExperimental::class)
```

Comment:
Since we have the file opt in at the top do we need this one here too?

Reply:
Removed it as it wasn't necessary in the file level 9f36407